### PR TITLE
added random to isawthat.js 

### PR DESCRIPTION
**Pull request by MrJay-exe**

### CODE REVIEW

File: events/iSawThat.js [hunk 1]
```diff
@@ -1,11 +1,16 @@
-const ignored = ['864311867765686332', '369212248553422850']
+const ignored = ['864311867765686332', '369212248553422850','609330932431716353']
```
Comment (vijaystroup) [inline events/iSawThat.js]: only leave the id of `864311867765686332` in the ignored list.

File: events/iSawThat.js [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
 
 const ISawThat = {
   name: 'messageDelete',
   async execute(message) {
     if (message.author.bot) return
+
+    const coin = (Math.floor(Math.random() * 2) == 0)
+
+    if(coin){
```
Comment (vijaystroup) [inline events/iSawThat.js]: format like `if (coin) {`

File: events/iSawThat.js [hunk 3]
```diff
@@ -1,11 +1,16 @@
...
     if (ignored.includes(message.author.id)) return
```
Comment (vijaystroup) [inline events/iSawThat.js]: indent line 11 and 12 with 2 spaces.

**Review by vijaystroup — CHANGES_REQUESTED** [review]

**Comment by MrJay-exe** [comment]
Alright changed it with the latest commit

File: events/iSawThat.js [hunk 4]
```diff
@@ -1,11 +1,16 @@
...
   name: 'messageDelete',
   async execute(message) {
     if (message.author.bot) return
-    if (ignored.includes(message.author.id)) return
-    await message.channel.send(`i saw that <@${message.author.id}>`)
+
+    const coin = (Math.floor(Math.random() * 2) == 0)
+
+    if (coin) {
+      if (ignored.includes(message.author.id)) return
```
Comment (vijaystroup) [inline events/iSawThat.js:11]: Put the ignored check after the bot check so in the case it is an ignored user, we don't perform the coin computation and just return from the function.

**Review by vijaystroup — CHANGES_REQUESTED** [review]

File: events/iSawThat.js (hunk 4 above)
Comment (VijayStroup) [reply events/iSawThat.js:11]: updates? 👀

**Comment by vijaystroup** [comment]
closing due to breaking change now.

**Pull request closed without merging**